SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -544,7 +544,9 @@ void CFVMFlowSolverBase<V, R>::ComputeUnderRelaxationFactor(const CConfig* confi
       /* We impose a limit on the maximum percentage that the
        density and energy can change over a nonlinear iteration. */
 
-      if ((iVar == 0) || (iVar == nVar - 1)) {
+      unsigned short tmp = 1; //DELETE ME TODO
```

Comment:
I wanted to generalize this so NEMO can be used.  This currently only works for a single species gas.